pulp_2to3_migration/app/pre_migration.py

```diff
@@ -53,13 +53,14 @@
 
             content_model = ContentModel(pulp2=pulp2_content_model,
                                          pulp_2to3_detail=pulp_2to3_detail_model)
-            pre_migrators.append(pre_migrate_content(content_model))
+            pre_migrators.append(pre_migrate_content(content_model,
+                                                     plugin.migrator.mutable_content_models))
```

Comment:
It just crossed my mind that we might not need to carry all the mutable models but just pass an argument whether the content we are migrating is mutable or not.

E.g.
```
mutable_type = content_model.pulp2.TYPE_ID in mutable_content_models
pre_migrators.append(pre_migrate_content(content_model, mutable_type)
```

Not critical or blocking, just sharing what I just noticed :)